### ISSUE TITLE
Nicht gemahnte Defizite

### ISSUE THREAD

**Issue by Raffenberg** [SVWS-NRW/Schild3-BetaTest#171]
In den letzten Jahren kam immer wieder der Wunsch auf, die Zählung von nicht gemahnten Defiziten automatisiert anhand des Jahrgangs zu setzen. Momentan gibt es in der Verwaltung ein Auswahlfeld, welches man manuell ein und ausschalten muss, wenn man die Versetzungsberechung durchführt. Wäre es nicht gut, wenn man dieses Feld in der Jahrgangstabelle aufnehmen würde, so dass man ENDLICH eine jahrgangsindividuelle Einstellung hat?

**Comment by JuergenRichter** [comment]
Das setzt aber auch eine entsprechende Ergänzung in der Datenbankstruktur voraus, das müsste dann Herr Pfotenhauer entscheiden

**Comment by FPfotenhauer** [comment]
Sollten wir machen, wenn Schild3.0 auf dem "Markt" ist. Wenn wir das einfach bei der Migration machen, dann werden die User das nicht mitbekommen und wundern sich dann.

**Comment by FPfotenhauer** [comment]
Wird mit Schild3.0 kommen.

Da wäre eine Auflistung der Schulformen und Jahrgängen und ggf. SGL gut:

1) GY > G8 > 09
2) GY > G9 > 10
3) RS > 10

usw......

**Comment by WJohannsen** [comment]
An RS und GY (G9) gibt es am Ende der Klasse 9 bereits den ESA als Abschluss, sodass auch hier sämtliche Defizite, auch nicht-gemahnte, zählen müssten.

§ 7 (4) APO-SI: " Ist mit der Versetzung der Erwerb eines Abschlusses [...] verbunden, werden bei der Entscheidung über die Versetzung und die Vergabe des Abschlusses oder der Berechtigung auch Minderleistungen berücksichtigt, die nicht abgemahnt worden sind."
§40 (4) APO-SI: "Eine Schülerin oder ein Schüler der Realschule, des Gymnasiums [...] erwirbt am Ende der Klasse 9 mit der Versetzung den Ersten Schulabschluss."

**Comment by Raffenberg** [comment]
Ja, auch in der 9 werden alle nicht gemahnten Defizite bei der Versetzung berücksichtigt. Da gab es auch ein Schreiben der Bezirksregierung zu.

**Comment by Raffenberg** [comment]
Wiedervorlage
